asset/css/markbind.css

```diff
@@ -25,6 +25,31 @@ pre > code.hljs {
     counter-reset: line;
 }
 
+pre > code.hljs[heading] {
+    border-top-right-radius: 0;
+}
+
+.code-block {
+    position: relative;
+}
+
+.code-block > .code-block-heading {
```

Comment:
`.code-block-heading` sounds specific enough without having to further qualify it with `.code-block >`. Let's remove the unnecessary qualifiers.